src/library/trackmodel.h

```diff
@@ -49,41 +49,48 @@ class TrackModel {
     };
     typedef int CapabilitiesFlags; /** Enables us to do ORing */
 
+    // Note that these enum values are used literally by controller scripts and must never be changed!
+    // Both reordering or insertion of new enum variants is strictly forbidden!
+    // New variants must always be inserted between the last valid and before the terminating variant SORTCOLUMN_ID_MAX!
     enum SortColumnId {
```

Comment:
Can you make this an enum class and use camelCase for the enum members?

Reply:
I tried to use **enum class SortColumnId : int {** instead of **enum SortColumnId {** but that requires a **static_cast<< int >>** for every single place where it's used. Is that what you intend?

Reply:
Yes.

Reply:
But isn't type saveness the reason for an enum class? If I use it everywhere wit static_cast, there is no type saveness at all...

Reply:
Scoping is also another reason for it.

Reply:
Done!